recipes/pybinding/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pybinding" %}
+{% set version = "0.8.1.dev0" %}
+{% set sha256 = "164f0e8be427dfe1d33dfea1a8b496fbfe1b88e9170ec2d1f40a372ba29811c9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dean0x7d/pybinding.git
+  git_rev: 1eb76d308e48fead39051ee687b0a2aec39d4409
+
+build:
+  skip: True  # [py<34 or osx]
```

Comment:
I included OS X on my initial commit, but the build on Travis is using the `beta-xcode6.1` image and it's not really cooperating: [clang crashed with a segfault](https://travis-ci.org/conda-forge/staged-recipes/jobs/173785686#L666). Not sure if it's because of the `beta` part or because of the `6.1` part. It compiles fine with `xcode7.x` images. Any chance of getting a newer image up on Travis?